sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -26,40 +28,313 @@ class ConfigurationSetting(Model):
     :param tags:
     :type tags: dict[str, str]
     """
+
     _attribute_map = {
-        'etag': {'key': 'etag', 'type': 'str'},
-        'key': {'key': 'key', 'type': 'str'},
-        'label': {'key': 'label', 'type': 'str'},
-        'content_type': {'key': 'content_type', 'type': 'str'},
-        'value': {'key': 'value', 'type': 'str'},
-        'last_modified': {'key': 'last_modified', 'type': 'iso-8601'},
-        'read_only': {'key': 'read_only', 'type': 'bool'},
-        'tags': {'key': 'tags', 'type': '{str}'},
+        "etag": {"key": "etag", "type": "str"},
+        "key": {"key": "key", "type": "str"},
+        "label": {"key": "label", "type": "str"},
+        "content_type": {"key": "content_type", "type": "str"},
+        "value": {"key": "value", "type": "str"},
+        "last_modified": {"key": "last_modified", "type": "iso-8601"},
+        "read_only": {"key": "read_only", "type": "bool"},
+        "tags": {"key": "tags", "type": "{str}"},
     }
 
     def __init__(self, **kwargs):
         super(ConfigurationSetting, self).__init__(**kwargs)
-        self.etag = kwargs.get('etag', None)
-        self.key = kwargs.get('key', None)
-        self.label = kwargs.get('label', None)
-        self.content_type = kwargs.get('content_type', None)
-        self.value = kwargs.get('value', None)
-        self.last_modified = kwargs.get('last_modified', None)
-        self.read_only = kwargs.get('read_only', None)
-        self.tags = kwargs.get('tags', None)
+        self.key = kwargs.get("key", None)
+        self.label = kwargs.get("label", None)
+        self.value = kwargs.get("value", None)
+        self.etag = kwargs.get("etag", None)
+        self.content_type = kwargs.get("content_type", None)
+        self.last_modified = kwargs.get("last_modified", None)
+        self.read_only = kwargs.get("read_only", None)
+        self.tags = kwargs.get("tags", {})
 
     @classmethod
-    def _from_key_value(cls, key_value):
+    def _from_generated(cls, key_value):
         # type: (KeyValue) -> ConfigurationSetting
         if key_value is None:
             return None
+        if key_value.content_type is not None:
+            if key_value.content_type.startswith(
+                FeatureFlagConfigurationSetting.feature_flag_content_type
+            ):
+                return FeatureFlagConfigurationSetting._from_generated(  # pylint: disable=protected-access
```

Comment:
What if we fail to interpret the value into a FeatureFlagConfigurationSetting? Should we just return an instance of ConfigurationSetting?

Reply:
I will add this in a try/except wrapper